src/main/java/org/skife/jdbi/v2/BeanMapper.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2004 - 2014 Brian McCallister
+ * Copyright (C) 2004 - 2015 Brian McCallister and others
```

Comment:
I'm not sure either.  It's just something I noticed, I don't have any reason to insist a change, just feels weird to have it inconsistent